staging-installcheck.py

```diff
@@ -182,7 +153,7 @@ def staging(self, project, force=False):
             return True
 
         repository_pairs = repository_path_expand(api.apiurl, project, repository)
-        staging_pair = [project, repository]
+        [project, repository]
```

Comment:
what is this suppose to do?

Reply:
that's what happens if you let a tool remove unused variables - it creates unused statements :)

I'll remove